packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -328,19 +328,23 @@ public void receiveCommand(
 
         if (!args.isNull(1)) {
           String text = args.getString(1);
-          reactEditText.maybeSetTextFromJS(getReactTextUpdate(text, mostRecentEventCount));
+          reactEditText.maybeSetTextFromJS(getReactTextUpdate(reactEditText, text, mostRecentEventCount));
```

Comment:
why this change? the break strategy seems to get updated through other pipelines

Reply:
`getTextUpdate` method was returning a new ReactTextUpdate object with hardcoded break strategy value of `0`. This was value getting copied over with each onChangeText event there by resetting breakStrategy to `0` even when other strategies were used in code.